config/schemas/config.yaml

```diff
@@ -1363,6 +1372,727 @@ properties:
         items:
           type: string
     type: object
+  calicoAccountant:
+    additionalProperties: false
+    properties:
+      backend:
+        default: iptables
+        enum:
+          - iptables
+          - nftables
```

Comment:
When rendering to markdown we want to add this `meta:enum`. Could you add the them? Do not spend to much time writing a description for them :slightly_smiling_face:  
```suggestion
        enum:
          - iptables
          - nftables
        meta:enum:
          iptables: "Description of iptables"
          nftables: "Description of nftables"
```

Reply:
Is this something from the new docs work? Don't think I have seen this way of documenting enum values. I would have expected a `description: "iptables dolor sit amet\n\nnftables lorem ipsum"` or similar

Reply:
Yes, it is from the new docs work :slightly_smiling_face: 
It will look like this in the end :slightly_smiling_face: 
![image](https://github.com/elastisys/compliantkubernetes-apps/assets/12862587/73fc2556-609d-40d0-b0fb-77f4d3d20b37)

Reply:
Looks amazing! Added some words paraphrased from Wikipedia and from comments in `common-config.yaml` for now.